src/api/app/models/token.rb

```diff
@@ -17,6 +17,9 @@ class Token < ApplicationRecord
 
   include Token::Errors
 
+  scope :owned_tokens, ->(user) { where(user: user).where.not(type: ['Token::Rss']).includes(package: :project) }
+  scope :shared_tokens, ->(user) { user.workflow_tokens.where.not(type: ['Token::Rss']).includes(package: :project) }
```

Comment:
`user.workflow_tokens` only returns instances of `Token::Workflow`, so it makes no sense to add `where.not(type: ['Token::Rss'])`